scripts/installAds.ts

```diff
@@ -0,0 +1,54 @@
+import { join } from 'https://deno.land/std@0.224.0/path/mod.ts'
+
+const projectPath = Deno.env.get('ADALO_APP_PROJECT_PATH') as string
+const FALLBACK_ID = 'ca-app-pub-1111111111111111~1111111111'
+
+const readJsonFile = async (path: string) => {
```

Comment:
I believe this can be replaced with an import:

```
const module = await import("./version.json", {
  with: { type: "json" },
});

Reply:
Ah, hadn't realize dynamic imports are supported in `deno` that way. That's col, I'll change it.